pkg/process/procutil/process_windows_toolhelp.go

```diff
@@ -240,14 +241,20 @@ func (cp *cachedProcess) fillFromProcEntry(pe32 *w32.PROCESSENTRY32) (err error)
 	if usererr != nil {
 		log.Debugf("Couldn't get process username %v %v", pe32.Th32ProcessID, err)
 	}
-	var cmderr error
 	cp.executablePath = winutil.ConvertWindowsString16(pe32.SzExeFile[:])
-	commandParams, cmderr := winutil.GetCommandParamsForProcess(cp.procHandle, false)
-	if cmderr != nil {
-		log.Debugf("Error retrieving full command line %v", cmderr)
-		cp.commandLine = cp.executablePath
-	} else {
-		cp.commandLine = commandParams.CmdLine
+	// set default commandLine value incase we can't read the actual commandLine due to the proteced process
```

Comment:
```suggestion
	// set default commandLine value in case we can't read the actual commandLine due to the protected process
```

Suggestion: I don't think this comment is necessary, can remove